test/blackbox-tests/test-cases/install-dry-run/run.t

```diff
@@ -36,24 +36,14 @@
 
   $ dune uninstall --dry-run 2>&1 | sed 's#'$(opam config var prefix)'#OPAM_PREFIX#'
   Removing (if it exists) OPAM_PREFIX/lib/mylib/META
-  Removing directory (if empty) OPAM_PREFIX/lib/mylib
```

Comment:
I haven't read this PR in detail, but why is this test changing? it seems unrelated.

Reply:
Indeed, it's an unrelated improvement. I submitted a separate PR for it: https://github.com/ocaml/dune/pull/2234